File: js/day69.js
const crypto = require('crypto');
const readline = require('readline');
const fs = require('fs');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter password: ', (password) => {
  rl.close();

  // Generate SHA-1 Hash
  const hash = crypto.createHash('sha1').update(password).digest('hex');

  // Save to txt file
  fs.writeFile('password_hash.txt', hash, (err) => {
    if (err) throw err;
    console.log('SHA1 Hash successfully saved to password_hash.txt');
  });
});

File: python/day69.py
import hashlib
import getpass

password = getpass.getpass("Enter a password: ")

# Convert string to bytes and generate the SHA-1 hexadecimal hash
sha1_hash = hashlib.sha1(password.encode('utf-8')).hexdigest()

# Save the resulting has into a txt file
output_file = "password_hash.txt"
with open(output_file, "w", encoding="utf-8") as file:
  file.write(sha1_hash)

print(f"Success! The SHA-1 hash has been written to {output_file}")